{"uuid": "0dabf58f-60bd-400c-8529-60b313afb892", "children": ["be1b879d-f9a1-469a-afe2-302f891cd653"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "TypeError: __init__() got an unexpected keyword argument 'timeout'\n", "trace": "  File \"/home/nikita/.local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/nikita/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/nikita/.local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/nikita/candela_projects/6052_branch_check/wlan-testing/tests/e2e/advanced/conftest.py\", line 75, in setup_profiles\n    return_var = fixtures_ver.setup_profiles( request, param, run_lf, instantiate_profile, get_configuration, get_markers)\n  File \"/home/nikita/candela_projects/6052_branch_check/wlan-testing/tests/fixtures_3x.py\", line 68, in setup_profiles\n    instantiate_profile_obj = instantiate_profile(controller_data=get_configuration['controller'], timeout=\"10\")\n"}, "start": 1645014405587, "stop": 1645014420229}], "start": 1645014405587, "stop": 1645014420363}